### ISSUE TITLE
failiure to run recovery menu

### ISSUE THREAD

**Issue by asdfghjkl403622** [GaryOderNichts/udpih_nxpayload#4]
Hello! 

Sorry for the non-descriptive title, I'm not really sure what the issue _is_.
I have bought a Japanese  Wii U off of ebay to cut costs, and intended to region change it to an american console for which there is a helpful [guide](https://lazr1026.github.io/regionchange/#/). However, after going through all the steps up until running the recovery menu using my switch, it fails. 
Here are the things I have checked:

- No other usb devices are plugged in
- I'm using the front usb ports
- I have tried fully removing power from the console before attempting to run the payload
- The firmware is version 5.5.X
- setting the boot loader to not auto boot for a better timing window
- I ran a checksum on the recovery menu for a sanity check
- I've tested this cable on multiple other applications, and it has no issues

To clarify exactly what happens when I make an attempt, I have to run the payload as soon as the **Nintendo** logo shows up in order to get past stage 2 with no errors. It then runs through, and the Wii U does not react at all. Nothing on the screen, and no changes in the LED. Any other timings or configurations and it fails at some step of the operation or freezes. 

This issue has eluded everyone I've talked to, so I'm hoping that this issue can be resolved here so anyone in the future can have a solution for this problem.

**Comment by ma-shulgin** [comment]
Hey, @asdfghjkl403622 ! I have absolutely the same issue. Have you managed to resolve it in some way?

**Comment by asdfghjkl403622** [comment]
nope, and I was told I would probably never get a response. This seems to be the case.
